src/Engine/Army.java

```diff
@@ -301,6 +300,24 @@ public GameEventQueue receiveBattleEvent(final BattleSummary summary)
           }
         }
           break;
+        case Persistent:
+        {
+          // As persistent tags is meant to be mostly a sidegrade, give half charge to the primary, and split the rest among the rest.
```

Comment:
Is this what the games do? It seems like this would basically be a downgrade, since having two (or more) COs just means you get to use powers less frequently. A single CO would be able to use his much more often.

Reply:
No games have persistent tags in them.
Cart/AWBW behavior is that the second CO gets half the primary's charge.

It *is* a downgrade for power timing... which is the intent. The intent is to balance persistent tags vs a single CO, though obviously the error bars here are pretty big.
In trade, you get a lot:
Flexibility to mix and match different D2Ds as you see fit.
The ability to stack different COs' D2Ds and powers.
The ability to hold more charge than a single CO can.
More powers to choose from.

Flexibility is power.